### ISSUE TITLE
snowman project was recently erased

### ISSUE THREAD

**Issue by mborgerson** [decompiler-explorer/decompiler-explorer#94]
[snowman](https://github.com/yegord/snowman) maintainer recently closed all open PRs and blanked the repo, apparently for [license related issues](https://github.com/yegord/snowman/pull/228#issuecomment-1460582712). My [fork](https://github.com/mborgerson/snowman) has a pre-scrub archive. This means for Decompiler Explorer, the snowman build source repo needs to either be updated, or the tool needs to be disabled, or removed entirely.

A quick read of the license issue in question seems to just be stating that the original smartdec source was modified, so I don't see an issue with just updating the source link and continuing to provide snowman as an available tool in Decompiler Explorer. Perhaps the maintainer will change their mind and resume the project at some point and we can move the link back.

As an aside, DE should provide access to older versions of the tools (#19) and handle the eventuality of some projects being abandoned or retired, and for open source projects, their source disappearing. I guess for this case we can just fork a repo archive into the org.

**Comment by CouleeApps** [comment]
... except it wasn't a license issue because they fulfilled the obligations of the GPL? https://github.com/yegord/snowman/blob/6fed71ccb650ebd1db1141676e1af8166835819e/doc/licenses.asciidoc

**Comment by mborgerson** [comment]
> ... except it wasn't a license issue because they fulfilled the obligations of the GPL? https://github.com/yegord/snowman/blob/6fed71ccb650ebd1db1141676e1af8166835819e/doc/licenses.asciidoc

Yes @CouleeApps, I don't think there is an actual violation here

**Comment by psifertex** [comment]
Agreed. Super not clear what exactly that (likely bot) is claiming is wrong. I've never heard a GPL interpretation that required you to specify every file changed especially when said files _are in source control and the info is redundant anyway_.

**Comment by mborgerson** [comment]
Snowman source was updated in 2523510415181592409224926044579584420260

We probably should just move the repo over to decompiler-explorer namespace for archival, but it's fine for now so I'll close the issue.
